graphql/schemabuilder/pagination.go

```diff
@@ -907,6 +907,7 @@ func (sb *schemaBuilder) buildPaginatedField(typ reflect.Type, m *method) (*grap
 		Type:           retType,
 		ParseArguments: argParser.Parse,
 		Expensive:      c.hasContext,
+		External:       true,
```

Comment:
maybe use Internal or Cheap instead?

(random question: would we benefit at all from codegen-ing scalar field accessors?)

Reply:
Codegening scalar accessors would definitely be advantageous, not sure if the scale would matter that much yet since we have some lower hanging fruit.